doc/installation/system/inifile.rst

```diff
@@ -226,13 +226,17 @@ privacyIDEA Nodes
 -----------------
 
 privacyIDEA can run in a redundant setup. For statistics and monitoring purposes you
```

Comment:
Not only for statistics and monitoring purposes, but please also add the purpose of node specific configuration.

Reply:
Ah, ok, this is only in the future pull requests. So at this stage this is actually correct.